spec/index.md

```diff
@@ -244,9 +244,11 @@ KL27 storage memory layout:
 ↓ KL27 flash address 0x20000                                ↓ KL27 flash address0x40000
 ↓         ↓ KL27 flash address 0x20400                      ↓
 ┌---------┬-------------------------------------------------┐
-|[config] |[data                           [ DAL’s config ]]|
+| KL27    | storage data                                    |
+| config  |[ file.ext -------------------- ][ DAL’s config ]|
 └---------┴-------------------------------------------------┘
-          ↑ storage address 0x0000         ↑ storage address set by file size
+          ↑ storage address 0x0000          ↑ address set by file size
+                 ↑ base64 start address (anywhere inside file.ext range)
```

Comment:
FYI: also mentions base64 here

Reply:
Updated the diagram to include both pointers, thanks Joe!